test/e2e/kubevirt.go

```diff
@@ -534,6 +542,22 @@ iperf3 -t 0 -c %[1]s -p %[2]d --logfile %[3]s &
 				Expect(err).NotTo(HaveOccurred())
 				return vmi.Status.MigrationState.Completed
 			}).WithPolling(time.Second).WithTimeout(20*time.Minute).Should(BeTrue(), "should complete migration")
+			Eventually(func() bool {
+				err := crClient.Get(context.TODO(), crclient.ObjectKeyFromObject(vmi), vmi)
+				Expect(err).NotTo(HaveOccurred())
+				if vmi.Status.MigrationState.SourcePod == "" {
+					return false
+				}
```

Comment:
You don't need this at eventually, at this state the migratiopn is finished so you should have it tall for sure.